### PR TITLE
docs: Encrypted storage

### PR DESCRIPTION
**Pull request by DRK3**
Signed-off-by: Derek Trider <Derek.Trider@securekey.com>

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

File: docs/encrypted_storage.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Aries Encrypted Storage
+
+The Aries framework can be figured to use encrypted storage. A concrete [EDV](https://identity.foundation/confidential-storage/) implementation is provided, but custom encrypted storage solutions can be injected.
```
Comment (llorllale) [inline docs/encrypted_storage.md:3]: s/implementation/client/

**Review by llorllale — APPROVED** [review]

File: docs/encrypted_storage.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+For example:
+```
+memProvider := mem.NewProvider()
+
+// Create the FormattedProvider with the WithCacheProvider option.
+formattedProvider, err := formattedstore.NewFormattedProvider(edvRESTProvider, encryptedFormatter, false, WithCacheProvider(memProvider))
+```
+
+## EDV Extensions (Optional Features)
```
Comment (troyronda) [inline docs/encrypted_storage.md:36]: Is there an issue created in the edv spec repo for these?

Comment (DRK3) [reply docs/encrypted_storage.md:36]: Yes

Comment (troyronda) [reply docs/encrypted_storage.md:36]: Can you provide the links, please. @DRK3

File: docs/encrypted_storage.md (hunk 1 above)
Comment (troyronda) [reply docs/encrypted_storage.md:3]: yes - should clarify that a client implementation is provided.

Comment (DRK3) [reply docs/encrypted_storage.md]: Changed the wording. Take a look and let me know if it sounds good.

File: docs/encrypted_storage.md (hunk 2 above)
Comment (DRK3) [reply docs/encrypted_storage.md:36]: @troyronda Sure:

Batch: https://github.com/decentralized-identity/confidential-storage/issues/138
Returning full documents on query: https://github.com/decentralized-identity/confidential-storage/issues/137

Comment (troyronda) [reply docs/encrypted_storage.md:36]: I also think references in the document would be helpful? @DRK3

Comment (DRK3) [reply docs/encrypted_storage.md:36]: @troyronda Done - take a look at the updated doc and let me know if it looks ok

**Comment by troyronda** [comment]
@DRK3 @llorllale @fqutishat  are you planning to link to this page from other docs pages?

**Comment by DRK3** [comment]
@troyronda @fqutishat @llorllale Just added a link to `project_structure.md`

File: docs/encrypted_storage.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# Aries Encrypted Storage
+
+The Aries framework can be figured to use encrypted storage. [An implementation conforming to the Store interface](../pkg/storage/store.go) that is capable of storing to an [EDV](https://identity.foundation/confidential-storage/) server is included, but custom implementations can be injected if desired.
+
+## Configuring Aries Framework with EDV storage in Go Code
+```
+// Create an edv.EncryptedFormatter with the necessary key data and crypto data to handle the conversion between unencrypted and encrypted data.
+// Check the unit tests to see an example of how you can create the jweEncrypter, jweDecrypter, and macCrypto objects.
```
Comment (troyronda) [inline docs/encrypted_storage.md:8]: A link to the example would be helpful.

**Review by llorllale — APPROVED** [review]

File: docs/encrypted_storage.md (hunk 3 above)
Comment (DRK3) [reply docs/encrypted_storage.md]: Done

File: docs/encrypted_storage.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# Aries Encrypted Storage
+
+The Aries framework can be figured to use encrypted storage. [An implementation conforming to the Store interface](../pkg/storage/store.go) that is capable of storing to an [EDV](https://identity.foundation/confidential-storage/) server is included, but custom implementations can be injected if desired.
```
Comment (troyronda) [inline docs/encrypted_storage.md:3]: "can be figured" typo

Comment (DRK3) [reply docs/encrypted_storage.md]: Fixed.

**Pull request merged**